ios/SSLPinning.mm

```diff
@@ -84,23 +108,17 @@ +(void)runChallenge:(NSURLSession *)session
   completionHandler:(void (^)(NSURLSessionAuthChallengeDisposition disposition, NSURLCredential *credential))completionHandler
 {
   NSString *host = challenge.protectionSpace.host;
-
-  // Read the clientSSL info from MMKV
-  __block NSString *clientSSL;
-  SecureStorage *secureStorage = [[SecureStorage alloc] init];
-
-  // https://github.com/ammarahm-ed/react-native-mmkv-storage/blob/master/src/loader.js#L31
-  NSString *key = [secureStorage getSecureKey:[self stringToHex:@"com.MMKV.default"]];
   NSURLCredential *credential = [NSURLCredential credentialForTrust:challenge.protectionSpace.serverTrust];
 
-  if (key == NULL) {
+  // Read the clientSSL info from MMKV using MMKVBridge
+  MMKVBridge *mmkvBridge = [self getMMKVInstance];
+  
+  if (!mmkvBridge) {
     return completionHandler(NSURLSessionAuthChallengePerformDefaultHandling, credential);
   }
 
-  NSData *cryptKey = [key dataUsingEncoding:NSUTF8StringEncoding];
-  MMKV *mmkv = [MMKV mmkvWithID:@"default" cryptKey:cryptKey mode:MMKVMultiProcess];
```

Comment:
We used to open MMKV directly and without having to care about path.
We should aim to keep it that way for simplicity.